show_notes/Episode_18_Part1.md

```diff
@@ -0,0 +1,18 @@
+# Episode 18, Part 1: Android Everywhere
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+* [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/92)
+
+In the first part of episode 18 Artur, Hannes und Artem discuss about android tablets, Chromebooks and Wear OS by Google.
+
+#### Hosts:
+
+- Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), [personal blog](http://artemzin.com), [GitHub](https://github.com/artem-zinnatullin)
+- Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe), [personal blog](http://hannesdorfmann.com), [GitHub](https://github.com/sockeqwe)
+- Artur Dryomov [@arturdryomov](https://twitter.com/arturdryomov), [personal blog](http://arturdryomov.online/), [GitHub](https://github.com/ming13)
```

Comment:
`https://arturdryomov.online/`. Actually I’m not really sure about HTTPS schemas since sane websites actually implement force redirect from HTTP to HTTPS. And it is kind of an overkill to use HTTPS for a blog anyway (except maybe analytics). @artem-zinnatullin, thoughts?